src/webapp/public/locales/en/translation.json

```diff
@@ -23,7 +23,14 @@
           "timer_fade_volume": "Fade volume",
           "toggle_output": "Toggle audio output",
           "sync_rfidcards_all": "Sync all audiofiles and card entries",
-          "sync_rfidcards_change_on_rfid_scan": "Change activation of 'on RFID scan'"
+          "sync_rfidcards_change_on_rfid_scan": "Change activation of 'on RFID scan'",
+          "next_song": "Next song",
+          "pause": "Pause",
+          "play": "Play",
+          "prev_song": "Previous song",
+          "shuffle": "Toggle Shuffle",
+          "repeat": "Toggle Repeat",
```

Comment:
"De-/Activate Repeat" or "Activate/Deactivate Repeat", like the description for repeat in line 155?

Reply:
Same as above. I took inspiration from other typical music player interfaces.